[2025-10-26T17:55:03.735+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-26T17:55:03.755+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flood_prediction_pipeline.predict_and_log scheduled__2025-10-26T17:50:00+00:00 [queued]>
[2025-10-26T17:55:03.762+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flood_prediction_pipeline.predict_and_log scheduled__2025-10-26T17:50:00+00:00 [queued]>
[2025-10-26T17:55:03.763+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-26T17:55:03.774+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): predict_and_log> on 2025-10-26 17:50:00+00:00
[2025-10-26T17:55:03.781+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'flood_prediction_pipeline', 'predict_and_log', 'scheduled__2025-10-26T17:50:00+00:00', '--job-id', '925', '--raw', '--subdir', 'DAGS_FOLDER/flood_prediction_dag.py', '--cfg-path', '/tmp/tmp_eey0b84']
[2025-10-26T17:55:03.781+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1122) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-26T17:55:03.783+0000] {standard_task_runner.py:91} INFO - Job 925: Subtask predict_and_log
[2025-10-26T17:55:03.783+0000] {standard_task_runner.py:63} INFO - Started process 1125 to run task
[2025-10-26T17:55:03.826+0000] {task_command.py:426} INFO - Running <TaskInstance: flood_prediction_pipeline.predict_and_log scheduled__2025-10-26T17:50:00+00:00 [running]> on host ac7c4fe30011
[2025-10-26T17:55:03.895+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flood_prediction_pipeline' AIRFLOW_CTX_TASK_ID='predict_and_log' AIRFLOW_CTX_EXECUTION_DATE='2025-10-26T17:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-26T17:50:00+00:00'
[2025-10-26T17:55:03.897+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-26T17:55:03.923+0000] {logging_mixin.py:188} INFO - ℹ️ Latest row already processed. Skipping prediction.
[2025-10-26T17:55:03.942+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-26T17:55:03.943+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-26T17:55:03.949+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=flood_prediction_pipeline, task_id=predict_and_log, run_id=scheduled__2025-10-26T17:50:00+00:00, execution_date=20251026T175000, start_date=20251026T175503, end_date=20251026T175503
[2025-10-26T17:55:03.998+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-26T17:55:04.014+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-26T17:55:04.018+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
